siriuspy/siriuspy/imgproc/main.py

```diff
@@ -0,0 +1,458 @@
+"""Module with main IOC Class."""
+
+import time as _time
+import logging as _log
+
+from pcaspy import Alarm as _Alarm
+from pcaspy import Severity as _Severity
+
+from .. import util as _util
+
+from .meas import Measurement
+
+
+class App:
```

Comment:
Please, change this name to something more explanatory.

Reply:
`meas`  module, `Measurement` class or `App` class? which one you mean. all are very generic.

I believe @RafaelLyra8  decided on these names based on what he saw in siriuspy `meas` subpackage.
But maybe rename `Measurement` class to something like `MeasBeamParams`  ou simply `MeasDVF`.

which name do you guys prefer?

Reply:
I was referring to the `App` class. The name `Measurement` doesn't bother me. But, from the suggestions you gave, I prefer `MeasDVF`.

Reply:
@fernandohds564 , many IOCs define this `App` class as it is nowadays (DiagSys/fofb/idff/imgproc/injctrl/machshift/posang/stabinfo). It is more likely we break "standard" by changing App in imgproc than keeping as it is. I suggest we leave this renaming of all `App`classes for a separate PR.

Reply:
Ok then.